projects/aca-shared/src/lib/services/aca-mobile-app-switcher.service.ts

```diff
@@ -113,8 +112,8 @@ export class AcaMobileAppSwitcherService {
     }
   }
 
-  clearSessionExpireTime(): void {
-    sessionStorage.removeItem('mobile_notification_expires_in');
+  removeClosingDialogTime(): void {
```

Comment:
this naming makes no sense, all the namings in the PR are going to confuse English speakers a lot, can be "clearSessionTimeout" or something similar

Reply:
changed to "clearSessionTimeout()" as suggested